route_following_plugin/include/route_following_plugin.h

```diff
@@ -80,18 +80,8 @@ namespace route_following_plugin
          * \param current_time Start time of the current maneuver passed as reference
          * \return A lane keeping maneuver message which is ready to be published
          */
-        cav_msgs::Maneuver composeManeuverMessage(double current_dist, double end_dist, double current_speed, double target_speed, int lane_id, ros::Time& current_time);
-        /**
-         * \brief Compose a stop and wait maneuver message based on input params
-         * \param current_dist Start downtrack distance of the current maneuver
-         * \param end_dist End downtrack distance of the current maneuver
-         * \param current_speed Start speed of the current maneuver
-         * \param start_lane_id Starting Lanelet ID of the current maneuver
-         * \param target_speed Target speed of the current maneuver
-         * \param current_time Start time of the current maneuver passed as reference
-         * \return A lane keeping maneuver message which is ready to be published
-         */
-        cav_msgs::Maneuver composeStopandWaitManeuverMessage(double current_dist, double end_dist, double current_speed, int start_lane_id, int target_lane_id, ros::Time& current_time, double end_time);
+        cav_msgs::Maneuver composeManeuverMessage(double current_dist, double end_dist, double current_speed, double target_speed, int lane_id, ros::Time current_time);
```

Comment:
If this method is just for lane following maneuvers as described by the comments it should be named as such.

Reply:
Having current_ in the parameter names is also a bit unclear as its not really the current state of the vehicle so much as the desired start state of the maneuver.